handle/middlewares/headers_test.go

```diff
@@ -0,0 +1,38 @@
+package middlewares
+
+import (
+	"github.com/stretchr/testify/require"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+)
+
+func TestAPIHeader(t *testing.T) {
+	url := "http://localhost:3000/test"
+	middleware := APIHeader(url)
+	recorder := httptest.NewRecorder()
+
+	router := middleware(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {}))
+	router.ServeHTTP(recorder, nil)
+
+	request := &http.Request{Header: http.Header{"Cookie": recorder.HeaderMap["Set-Cookie"]}}
```

Comment:
The linter identifies `HeaderMap` as deprecated. Maybe `recorder.Header()` is a good alternative?